### DISCUSSION TITLE
More settings for CC/Subtitles in the jellyfin client

### DISCUSSION THREAD

**Discussion by PICKLERICK2005** [jellyfin/jellyfin-desktop#618] [Ideas]
Hello, a small thing the player lacks and sometimes really irritates me is the lack of subtitle settings, so I thought I'd add my suggestion here :D

Options like:

- Subtitle scale
- font
- color
- subtitle (and maybe audio) delay settings

Scale is the most important one.
Thanks to all maintainers and contributors for such an amazing open source media management platform :)

**Comment by Dangonizer** [comment]
Per episode subtitle delay settings would be very appreciated. The webplayer already has this.
